lib/src/util/logging.cc

```diff
@@ -101,7 +101,6 @@ static void setupLoggingImp(std::ostream& log_stream,
         sink->set_filter(expr::has_attr(access_outcome));
         auto core = boost::log::core::get();
         core->add_sink(sink);
-
```

Comment:
That's not what I meant. With Boost 1.62, calling `new sink_t(std::move(access_stream))` fails because the argument forwarding doesn't work. I meant this should be changed to `auto sink = boost::make_shared<sink_t>(access_writer(std::move(access_stream)));`.